_posts/2024-06-20-pyconus-pyopensci-sprints.md

```diff
@@ -0,0 +1,358 @@
+and across the United States for our 1-day sprint. This resulted in over 35 issues and
+pull requests where volunteers helped fix issues on our website, in our tutorials, and
+in our packaging and peer review guidebooks. Many long-standing issues and bugs were
+fixed thanks to this wonderful Python community.
+
+We had so much activity that I'm still working on closing up issues and pull
+requests now—weeks after the sprint
+ended!
+
+After such a tremendous experience this year, I wanted to share with you the things
+that we've learned at pyOpenSci about running beginner-friendly sprints.
+
+## What is a sprint like?
+
+If you haven’t been to a sprint before, it’s an experience every open source
+enthusiast should have. Sprints are where people come together to work on a
+project. The word sprint can be used in a professional development setting. This might describe a team of developers working
+together to release a new feature. For the volunteer open source community, sprints often mean volunteers
+helping maintainers work on various parts of a project. And it's important to keep in mind that often the maintainers are
+also volunteers.
+
+### What do people sprint on for pyOpenSci?
+pyOpenSci is a non profit organization that is devoted to building diverse and
+inclusive community that supports making science more open and collaborative.
+Among other things that we do, we run [open peer review of scientific Python software](/about-peer-review/index.html).
+
+Running an volunteer-lead open peer review process for software is a challenge. As we grow,
+it requires infrastructure to support tracking the process. We also create
+community driven beginner-friendly
+ [Python packaging resources](https://www.pyopensci.org/python-package-guide/) and
+ [tutorials](https://www.pyopensci.org/python-package-guide/tutorials/intro.html).
+High quality learning resources also require maintenance and review.
+As such we always need a mixture of packaging experts and those newer to packaging
+to catch errors and points of confusion.
+
+As the Executive Director and Founder of pyOpenSci, I created most of our
+pyOpenSci GitHub infrastructure to support our mission. While I cherish times
+when I can work on technical things, it’s hard to keep up.
+Mission, vision and community work will always come first.
+
+I really appreciate when community members help us tick off open issues,
+big and small. These contributions propel the pyOpenSci mission of making science
+more open and collaborative forward.
+
+
+## Barriers to contribute to open source
+
+While sprints are a great way to engage the community in
+supporting an organization's (or a project's) mission, there are many barriers for contributors to consider.
+
+These barriers include:
+
+* Time to contribute.
+* Skills to contribute.
+* Confidence in skills / fear of contributing the wrong thing.
+* Privilege: This is a loaded one. Open Source can't be diverse if it requires privilege
```

Comment:
I think this section would benefit from a bit more explanation on what you mean by privilege.